File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        String str = Integer.toString(x);
        char[] charArr = str.toCharArray(); 
        int n = charArr.length;

        for(int i = 0; i < n/2; i++){
            if(charArr[i] != charArr[n-i-1]){
                return false;
            }
        }
        return true; 
    }
}

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        string = str(x)
        charArr = list(string)
        for i in range(int(len(charArr)/2)):
            if charArr[i] != charArr[len(charArr) - i - 1]:
                return False
        return True